packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-misc--to-es2015/leaked-context-get-metadata/exec.js

```diff
@@ -0,0 +1,15 @@
+let getMetadata;
+let metadataSymbol = Symbol();
+
+function decMethod(_, context) {
+  ({ getMetadata } = context);
+  getMetadata(metadataSymbol);
+}
+
+class C {
+  @decMethod m() {}
+}
+
+expect(() => {
+  getMetadata(metadataSymbol);
```

Comment:
Will it throw if `getMetadata` is called in a static field decorator? If it throws then maybe we'd better move the test to a static field decorator so we are more confident that the context method is not invokable after the method decorators are applied.

Reply:
It should throw in the very next decorator, so we could apply a second decorator to the same element and have it call it